jupyter_server_ai_tools/__init__.py

```diff
@@ -1,24 +1,12 @@
-from .extension import Extension
-from .tool_registry import (
-    find_tools,
-    parse_anthropic_tool_call,
-    parse_mcp_tool_call,
-    parse_openai_tool_call,
-    parse_vercel_tool_call,
-    run_tools,
-)
-
-__all__ = [
-    "find_tools",
-    "run_tools",
-    "parse_openai_tool_call",
-    "parse_anthropic_tool_call",
-    "parse_mcp_tool_call",
-    "parse_vercel_tool_call",
-]
+from .app import AiServerToolsApp
```

Comment:
Minor point: I would probably name it `AIServerToolsApp` as `AI` is always capitalized.